cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -9,10 +11,14 @@
 class TowerVariableValue(models.Model):
     _name = "cx.tower.variable.value"
     _description = "Cetmix Tower Variable Values"
+    _inherit = [
+        "cx.tower.reference.mixin",
```

Comment:
please, fix `_compute_variable_ids` method

Reply:
@GabbasovDinar what's wrong with it?

Reply:
@ivs-cetmix you need to compare these methods when the mixin is connected and when it is not. method uses the methods of the mixin, in one case by accessing the mixin model (because it is not connected) and in the second case using inherited methods, so, we need refactoring this method, because the mixin was connected with the model.